ivy_tests/test_ivy/test_functional/test_core/test_statistical.py

```diff
@@ -166,7 +166,41 @@ def test_mean(
         axis=axis,
         keepdims=keep_dims,
     )
+# median
```

Comment:
I dont think the frontend tasks require you to add tests for Ivy's API aswell

Reply:
oh i did not know that so I should remove the median tst?